src/lib.rs

```diff
@@ -9,18 +9,22 @@ mod memory;
 mod ovec;
 mod secret;
 mod symmetric_key;
+#[cfg(any(test, feature = "test-utils"))]
```

Comment:
Why do we need a `test-utils` feature?

Reply:
Generic tests are exported should  someone implement a custom memory. The in-memory back-end could be gated behind this feature too, instead of having an additional feature `bench`.